src/app/helpers/page-data-utils.js

```diff
@@ -99,7 +99,7 @@ export function useTextFromSlug(slug) {
 
                         setText(newDoc.body.innerHTML);
                         setHead({
-                            title: newDoc.head.querySelector('title').textContent,
+                            title: newDoc.head.querySelector('title')?.textContent,
```

Comment:
should this default to anything in case the queryselector doesnt return anything?

Reply:
There is a default when the title value is used.